_posts/0000-01-02-azaimirul.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
I have some kind of feeling insecure with the older generations such as early 90's and past. They're already faced the real-university life completely with the very happy-go-lucky ages when there's no smartphones or any other gadgets that separates the society's interactions compared to me that I've been in a beautiful lies just at the tips :''